src/renderer/components/+preferences/helm-charts.tsx

```diff
@@ -106,6 +104,7 @@ export class HelmCharts extends React.Component {
             formatOptionLabel={this.formatOptionLabel}
             controlShouldRenderValue={false}
             className="box grow"
+            themeName="lens"
```

Comment:
Created another theme for `<Select/>` to preserve it until all UI components will have unifying look. Now inputs in "settings" pages look different (intentionally) from same elements in the dashboard.